sites/partners/src/applications/PaperApplicationDetails/sections/DetailsTerms.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useContext } from "react"
+import { t, GridSection, ViewItem, GridCell } from "@bloom-housing/ui-components"
+import { DetailsApplicationContext } from "../DetailsApplicationContext"
+
+const DetailsTerms = () => {
+  const application = useContext(DetailsApplicationContext)
+
+  return (
+    <GridSection
+      className="bg-primary-lighter"
+      title={t("application.review.terms.title")}
+      inset
+      grid={false}
+    >
+      <GridCell>
+        <ViewItem label={t("application.details.signatureOnTerms")}>
+          {(() => {
+            if (application.acceptedTerms === null) {
+              return t("t.n/a")
+            } else if (application.acceptedTerms) {
```

Comment:
If acceptedTerms is "no" this will yield a translation for yes

Reply:
but "acceptedTerms" property is a boolean :) I tested 3 scenarios and everything is detected correctly.